release_images.yml

```diff
@@ -118,23 +110,23 @@ release_images:
       force_release: False
   9:
     framework: "tensorflow"
-    version: "1.15.3"
+    version: "1.15.4"
     training:
       device_types: ["cpu", "gpu"]
-      python_versions: ["py37", "py36", "py27"]
+      python_versions: ["py37"]
```

Comment:
Please remove "cpu" from `device_types`, or make it a separate entry, because this will apply `disable_sm_tag` to both.